### DISCUSSION TITLE
PDC for Nuendo on Mac

### DISCUSSION THREAD

**Discussion by snakenewton** [apohl79/audiogridder#1381] [Q&A]
Hi , 
I’m totally new to AG. I’ve used VE pro and Live Professor as plug hosts and both are a bit cumbersome for studio work.   I was wondering how Nuendo and AG work together with regard to Plug-in Delay Compensation ? Nuendo is essentially the same audio engine as Cubase so would be very happy to hear any Steinberg experiences 
Thanks 🙏🏻
Snake

**Comment by yoshimodular** [comment]
I use it locally everyday (same computer) in Mac, but I’m planning to add another machine. The only remarkable issue is that I have to export online to avoid some issues. What I do is just a Main Bus group routed to a track and I record the resulted audio.

AudioGridder is now a key piece in my studio and true live saver.
